### DISCUSSION TITLE
Web FW update Error 507 & OTA update not possible

### DISCUSSION THREAD

**Discussion by SpaceTeddy** [emsesp/EMS-ESP32#997] [General]
**Bug description**
Not possible to update the EMS-ESP32 firmware in Web interface (upload/download). OTA update with espota.py tool is not possible as well. Any idea to get the FW updated w/o using serial connection? thx

**Steps to reproduce**
Installed FW 3.5.0b8
'''EMS-ESP Version
v3.5.0b8

Device (Platform / SDK)
ESP32 / v4.4.2

System Uptime
000+00:07:29.085

CPU Frequency
160 MHz

Heap (Free / Max Alloc)
128 KB / 79 KB

Flash Chip (Size / Speed)
4.096 KB / 40 MHz

Application (Used / Free)
1.887 KB / 33 KB

File System (Used / Free)
20 KB / 172 KB'''

**Expected behavior**
update FW version to 

**Screenshots**
Error 507
<img width="928" alt="image" src="https://user-images.githubusercontent.com/9148964/209849380-80e31ffd-9fc6-4d55-b6e2-8f608a294a01.png">

OTA update:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/9148964/209849511-45fa0c57-54e7-42c9-9ccb-e57dcd968115.png">

**Comment by MichaelDvP** [comment]
Your ota partition is only 1920 kb, the filesystem 192 kb. You need to flash with ems-esp partiton table using USB and esptool or https://github.com/emsesp/EMS-ESP-Flasher.

**Comment by SpaceTeddy** [comment]
alright, I used the [EMS-ESP-Flasher](https://github.com/emsesp/EMS-ESP-Flasher) and it works out of the box. After reflashing, the Web update is functional.  
Thanks for giving the hint with the partitions.
-> closed
